controllers/secret.go

```diff
@@ -32,6 +32,7 @@ func (r *VolumeReplicationReconciler) getSecret(name, namespace string) (map[str
 	if err != nil {
 		if apierrors.IsNotFound(err) {
 			r.Log.Error(err, "Secret not found", "Secret Name", name, "Secret Namespace", namespace)
```

Comment:
```suggestion
			r.Log.Error(err, "secret not found", "Secret Name", name, "Secret Namespace", namespace)
```

lets keep every message in the same case (lower case) for consistency.